apps/haproxy/haproxy.cfg

```diff
@@ -70,7 +70,10 @@ frontend http
 		http-request capture req.hdr(Host) len 15
 		http-response lua.cors
 		# https://www.haproxy.com/blog/haproxy-log-customization/
-		log-format "%ci:%cp [%tr] %ft %b/%s %TR/%Tw/%Tc/%Tr/%Ta %ST %B %CC %CS %tsc %ac/%fc/%bc/%sc/%rc %sq/%bq %hr %hs %{+Q}r"
+		# https://www.haproxy.com/documentation/hapee/1-8r1/onepage/#8.2.4
+		# log-format "%ci:%cp [%tr] %ft %b/%s %TR/%Tw/%Tc/%Tr/%Ta %ST %B %CC %CS %tsc %ac/%fc/%bc/%sc/%rc %sq/%bq %hr %hs %{+Q}r"
```

Comment:
@ramirogm Can we delete the old log formats to keep the files a bit cleaner? We can always see the previous log formats through git.

Reply:
Sure, no need to keep it